lib/types.js

```diff
@@ -1285,7 +1285,7 @@ module.exports = function(expect) {
     inspect(value, depth, output) {
       return output.singleQuotedString(value);
     },
-    diffLimit: 4096,
+    diffLimit: 40960,
```

Comment:
My OCD votes for 32768 or 65536 😼

Reply:
Is your OCD old enough to vote? Let's do 65536 then.